pyactiveresource/activeresource.py

```diff
@@ -830,6 +832,7 @@ def save(self):
                 self.errors.from_xml(err.response.body)
             elif self.klass.format == formats.JSONFormat:
                 self.errors.from_json(err.response.body)
+            self.errors.code = err.code;
```

Comment:
the description of this method above leads me to think that this should only be called for ResourceInvalid errors and that network errors are handled elsewhere. Also can you add a test showing this working